src/optimisation/Optimisation.jl

```diff
@@ -1,150 +1,25 @@
 module Optimisation
 
 using ..Turing
+using ..Turing: NamedArrays, ModeResult, MLE, MAP, OptimLogDensity, OptimizationContext, ModeEstimator
+using Optimization
+using OptimizationOptimJL: LBFGS
 using Bijectors
 using Random
-using SciMLBase: OptimizationFunction, OptimizationProblem, AbstractADType, NoAD
+using SciMLBase: OptimizationFunction, OptimizationProblem, AbstractADType, NoAD, solve, AbstractSciMLAlgorithm
```

Comment:
How do you feel about qualifying also references to imports from the standard library? Or at least only importing from them only the things you need, e.g. `using Random: rand`, to avoid polluting the namespace with a bazillion things not explicitly named in our module anywhere.

Reply:
I think it sort of depends, e.g. `rand` is just used to much that we probably shouldn't bother qualifying it.

Similarly for `logpdf`, etc. Overall, we're probably happy with anything in `Base` being `using`ed in addition to Distributions.jl.